### ISSUE TITLE
Navigation button does not work as expected on Android Internet browser

### ISSUE THREAD

**Issue by TheRealWaldo** [n33/skel-layers#13]
In the default browser on all of my Android devices, the Navigation button doesn't behave properly; it will either not open, open and immediately click the first item, open and display whitespace, etc.

This issue occurs even on http://getskel.com/docs and baseline.  

It doesn't appear specific to any version of android.

**Comment by orugari** [comment]
I think it is simply need to add compatibility on the transform line because I think the problem is here. translation is done but transform.
need to add  -webkit-transform: and -ms-transform:
I just don't know how to.

**Comment by RedGossip** [comment]
https://gist.github.com/RedGossip/d794456480ac64d726e0
it solved  my problem..

**Comment by n33** [comment]
Alright folks, think I figured it out: tapping something with the `toggleLayer` action actually triggers its handler twice, resulting in the unpredictable behavior you're describing. It's most apparent on the AOSP browser, but I've actually been able to reproduce it on iOS with the right timing.

Anyhow, here's a dev version of Layers with a fix applied:

http://n33.co/downloads/tmp/skel-layers.min.js

Let me know if it works.
